### PR TITLE
yay this is my first commitgit status

### PR DESCRIPTION
**Pull request by bryanng77**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.uob</groupId>
+    <artifactId>uobcapstone</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+        <dependencies>
+    <!-- https://mvnrepository.com/artifact/com.mysql/mysql-connector-j -->
+    <dependency>
```
Comment (seshasai73) [inline pom.xml:13]: This is how comments are made
No changes are needed

Comment (bryanng77) [reply pom.xml:13]: thank you for the comment.

Comment (gavinyuenn) [reply pom.xml:13]: thank you for the reply.

**Review by seshasai73 — APPROVED** [review]
LGTM

**Review by bryanng77 — COMMENTED** [review]
code is satisfactory, good job. Approved.

**Review by gavinyuenn — COMMENTED** [review]
NOT APPROVED! >:)

**Pull request merged**